plugins/com.google.cloud.tools.eclipse.sdk.ui/src/com/google/cloud/tools/eclipse/sdk/ui/preferences/CloudSdkPreferenceArea.java

```diff
@@ -174,19 +173,23 @@ private void updateManagedSdk() {
 
   private void updateSelectedVersion() {
     String version = Messages.getString("UnknownVersion"); //$NON-NLS-1$
+    String location = null;
     if (!cloudSdkManager.isManagedSdkFeatureEnabled() || chooseSdk.getSelection()) {
-      Path path = Paths.get(sdkLocation.getStringValue());
+      location = sdkLocation.getStringValue();
+      Path path = Paths.get(location);
       version = getSdkVersion(path);
     } else if (cloudSdkManager.isManagedSdkFeatureEnabled()) {
       try {
         Path home = ManagedCloudSdk.newManagedSdk().getSdkHome();
         version = getSdkVersion(home);
+        location = home.toString();
       } catch (UnsupportedOsException ex) {
         // shouldn't happen but if it does we'll just leave
         // version set to Unknown
       }
     }
     sdkVersionLabel.setText(Messages.getString("SdkVersion", version)); //$NON-NLS-1$
+    sdkVersionLabel.setToolTipText(location);
```

Comment:
What happens if location is null here?

Reply:
Then there is no tooltip: setting null removes the tooltip.